components/mpas-ocean/src/shared/mpas_ocn_vmix.F

```diff
@@ -844,6 +849,16 @@ subroutine ocn_vel_vmix_tend_implicit_spatially_variable_mannings(forcingPool, b
          cell2 = cellsOnEdge(2,iEdge)
 
          ! average cell-based implicit bottom drag to edges and convert Mannings n to Cd
+         if (trim(config_thickness_drag_type) == 'harmonic-mean') then
```

Comment:
Is there a way to use `thickEdgeDragChoice` here?  If so, that would be preferable.  If there is an issue with circular dependencies, maybe @philipwjones has a suggestion.